cmd/contrib/scripts/main.go

```diff
@@ -0,0 +1,55 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/mitchellh/go-homedir"
+)
+
+// Retrieve the config directory to read or write to
+func main() {
+	opConfigDir, _ := os.LookupEnv("OP_CONFIG_DIR")
+	xdgConfigHome, _ := os.LookupEnv("XDG_CONFIG_HOME")
+	home, _ := homedir.Dir()
+
+	// This logic is based on the order of precedence outlined in the CLI documentation:
+	// https://developer.1password.com/docs/cli/config-directories
+	// Note: At the moment, if the --config flag is being used to specify the config
+	// directory then this script does not search in that location
```

Comment:
This script would only be executed in the `make plugin/build` command, where no `--profile flags` are to be set. Did you mean that this function could detect what `--profile` flag the user usually runs their `op` commands with? I think that is not something we could or should be looking at anyway. If a user keeps setting the `--profile flag` all the time, then they'll probably want to consider setting the `OP_CONFIG_DIR` env var anyway. 

So I think we could safely remove this Note:

Reply:
I elaborated on this in the `Additional Notes` section of the PR. It's related to the use of the `--config` flag to specify the config directory location, not `--profile`. But yep I agree that removing the Note should be safe, done!